app/templates/components/forms/admin/settings/system-form.hbs

```diff
@@ -44,7 +44,7 @@
     {{ui-radio label=(t 'Flask Filtering') checked='checked' name='search' disabled=true}}
   </div>
   <div class="field">
-    {{ui-radio label=(t 'Elastic Search (Coming Soon)') name='search' disabled=true value='elastic'}}
+    {{ui-radio label=(t 'Elastic Search') name='search' disabled=true value='elastic'}}
```

Comment:
Actually, remove the entire search choice section, we are not proceeding with elastic search in the foreseeable future.